externals/lua/src/Makefile

```diff
@@ -70,16 +70,16 @@ echo:
 # Convenience targets for popular platforms
 
 mingw:
-	$(MAKE) "CC=i686-w64-mingw32-gcc -std=gnu99" "AR=i686-w64-mingw32-ar rcu" \
```

Comment:
This will break the build for everyone outside the MSYS2. You should detect MSYS2 as environment and add run the build as bare Linux.

Reply:
Got it, I will work on it